test/test_interface.idl

```diff
@@ -40,9 +40,14 @@ interface TestInterface {
   attribute double doubleNumber;
   readonly attribute double readonlyDoubleNumber;
   static attribute double staticDoubleNumber;
-  double getDoubleAttribute();
   void readonlyAssignTest(double number);
   static double StaticTest(double number);
+  double getDoubleNumber();
```

Comment:
You already defined ```attribute double doubleNumber;``` but you also define getDoubleNumber() here.
This should be conflict. The ```attribute``` in WebIDL SHOULD be represented as Get/Set methods in C++ layer.